x-pack/test/functional/apps/painless_lab/painless_lab_flyout.ts

```diff
@@ -21,9 +37,24 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     it('click show API request button and flyout should appear in page', async () => {
+      // replace the default script with a simpler one
+      await monacoEditor.setCodeEditorValue(TEST_SCRIPT);
```

Comment:
Kinda seems odd to set smth here, and then not check that it indeed was set.

Reply:
There is the request_flyout.tsx that covers this. should I also add something here?